MediaWikiKit/MediaWikiKit/MWKSavedPageList.m

```diff
@@ -35,14 +43,18 @@ - (instancetype)initWithDataStore:(MWKDataStore*)dataStore {
     return self;
 }
 
-#pragma mark - Entry Access
-
-- (MWKSavedPageEntry*)entryAtIndex:(NSUInteger)index {
-    return [super entryAtIndex:index];
+- (void)importEntries:(NSArray*)entries {
+    NSArray<MWKSavedPageEntry*>* validEntries = [entries bk_reject:^BOOL (MWKSavedPageEntry* entry) {
+        return entry.title.text.length == 0;
+    }];
+    NSArray<MWKSavedPageEntry*>* uniqueValidEntries = [[NSOrderedSet orderedSetWithArray:validEntries] array];
```

Comment:
If NSOrderedSet encounters a duplicate - Which one does it keep? Do we care? Just curious if we know or if it even matters.